services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset.distinct()
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
```

Comment:
The return type of the `get_movie_by_id` function is incorrectly specified as `QuerySet`. It should be `Movie` since the function returns a single instance of `Movie`, not a queryset.